recipes/hepmc2/meta.yaml

```
@@ -0,0 +1,57 @@
+{% set name = "hepmc2" %}
+{% set version = "2.06.07" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: http://lcgapp.cern.ch/project/simu/HepMC/download/HepMC-{{ version }}.tar.gz
+  sha256: a0bdd6f36a3cc4cb59d6eb15cef9d46ce9b3739cae3324e81ebb2df6943e4594
+  folder: source
+  patches:
+    - patches/hepmc-2.03.06-reflex.patch
+    - patches/hepmc-2.06.07-WeightContainer-fix-size_type.patch
+    - patches/HepMC-2.06.07-nodoc.patch
+    - patches/HepMC-2.06.07-broken-tests.patch
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+  host:
+
+  run:
+
+test:
+  source_files:
+    - source/
+  requires:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+
+about:
+  home: http://hepmc.web.cern.ch/hepmc/
+  license: LGPL-2.1
+  license_family: LGPL
```

As most open source licences require a copy of the licence to be included this field should always be included:
```suggestion
  license_family: LGPL
  license_file: source/INSTALL
```
(tinyxml2 only needed a copy in the recipe directory as it's not included in their source tarball)